Realm/Realm/Realm.csproj

```diff
@@ -29,6 +29,10 @@
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.3" />
     <PackageReference Include="Remotion.Linq" Version="2.2.0" />
     <ProjectReference Include="..\Realm.Fody\Realm.Fody.csproj" PrivateAssets="None" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
We need to check if we can create JSON strings without adding new dependencies. In general this is not an issue, but it's better to try to reduce, especially if we use the dependency in very few places.